cli/operator/config/create.go

```diff
@@ -2,18 +2,25 @@ package config
 
 import (
 	"encoding/json"
+	"errors"
+	"math/big"
 	"os"
+	"regexp"
+	"strings"
 
 	eigensdkTypes "github.com/Layr-Labs/eigensdk-go/types"
+	"github.com/NethermindEth/eigenlayer/cli/prompter"
 	"github.com/NethermindEth/eigenlayer/internal/types"
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v3"
 )
 
-func CreateCmd() *cobra.Command {
+const zeroAddress = "0x0000000000000000000000000000000000000000"
```

Comment:
maybe just use it from eigensdk - https://github.com/Layr-Labs/eigensdk-go/blob/master/types/operator.go#L17